csv-collector/cmd/post_process.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020, Marcelo Jorge Vieira
+// Licensed under the AGPL-3.0+ License
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/labstack/gommon/log"
+	"github.com/olhoneles/politicos-go/csv-collector/collector"
+	"github.com/spf13/cobra"
+)
+
+var postProcessCmd = &cobra.Command{
+	Use:   "post-process",
+	Short: "Post-processes the imported data",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
We can improve the output when there is an error, try to run:

```
./collector post-process gsgsf
```

The output:
```json
{"time":"2020-11-08T14:00:16.208141253-03:00","level":"FATAL","prefix":"-","file":"post_process.go","line":"23","message":"step 'gsgsf' unrecognized."}
```

Using `RunE` instead of `Run`, we have:

```bash
Error: step 'gsgsf' unrecognized.
Usage:
  collector post-process [flags]

Flags:
  -h, --help   help for post-process

time="2020-11-08T13:58:48-03:00" level=error msg="step 'gsgsf' unrecognized."
```
The diff:

```diff
-       Run: func(cmd *cobra.Command, args []string) {
+       RunE: func(cmd *cobra.Command, args []string) error {
                subCmd := args[0]
                log.Debugf("Post-processing '%s'...", subCmd)
                err := runPostProcess(subCmd)
                if err != nil {
-                       log.Fatal(err)
+                       return err
                }
+               return nil
        },

```